kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -200,6 +202,37 @@ showLeafs = do
     showPair :: (NodeStates, [Graph.Node]) -> String
     showPair (ns, xs) = show ns <> ": " <> show xs
 
+showPrecBranch
+    :: MetaOrObject level
+    => Maybe Int
+    -> StateT (ReplState level) Simplifier ()
+showPrecBranch mnode = do
+    Strategy.ExecutionGraph { graph } <- Lens.use lensGraph
+    node <- Lens.use lensNode
+    let node' = maybe node id mnode
+    if node' `elem` Graph.nodes graph
+        then do
```

Comment:
No need for `do` if it's a single statement. Should also make it `then putStrLn' $ show ...`.